docs/operations/control_plane_migration.md

```diff
@@ -29,5 +29,5 @@ For controlplane migration, operators with the necessary RBAC can use the [`shoo
 ```bash
 export NAMESPACE=my-namespace
 export SHOOT_NAME=my-shoot
-kubectl get --raw /apis/core.gardener.cloud/v1beta1/namespaces/${NAMESPACE}/shoots/${SHOOT_NAME} | jq -c '.spec.seedName = "<destination-seed>"' | kubectl replace --raw /apis/core.gardener.cloud/v1beta1/namespaces/${NAMESPACE}/shoots/${SHOOT_NAME}/binding -f - | jq -r '.spec.seedName'
+kubectl get --raw /apis/core.gardener.cloud/v1beta1/namespaces/${NAMESPACE}/shoots/${SHOOT_NAME} | jq -c '.spec.seedName = "'$DESTINATION_SEED'"' | kubectl replace --raw /apis/core.gardener.cloud/v1beta1/namespaces/${NAMESPACE}/shoots/${SHOOT_NAME}/binding -f - | jq -r '.spec.seedName'
```

Comment:
This as well?

Reply:
... due to hurrying up I did a `git add.` .. let me remove those